libs/designer/src/lib/ui/panel/templatePanel/createWorkflowPanel/usePanelTabs.tsx

```diff
@@ -62,30 +65,56 @@ export const useCreateWorkflowPanelTabs = ({
     }
   }, [dispatch, mapping, existingWorkflowName, existingWorkflowNames, parametersExist, selectedTabId, kind]);
 
-  const handleCreateClick = useCallback(async () => {
-    setIsLoadingCreate(true);
-    await onCreateClick(() => setIsCreated(true));
-    setIsLoadingCreate(false);
-  }, [onCreateClick]);
+  const { isLoading: isCreating, mutate: createWorkflowFromTemplate } = useMutation(async () => {
+    setErrorMessage(undefined);
+    const logId = LoggerService().startTrace({
+      name: 'Create Workflow from Template',
+      action: 'createWorkflowFromTemplate',
+      source: 'usePanelTabs.tsx',
```

Comment:
Lets not use the file name for defining areas and source. Lets keep source as 'CreateTabs' and area as 'Templates.createTabs'

Reply:
subscriptionId => appName
- workflowAppName: